WordPress/gutenberg
# Image: Adds the block controls for uploading image.

## What?
Fixes #54867

## Why?
In small width size the image block placeholder looks broken and it's hard to choose an image from there.

## How?
This PR is adding a BlockControl to choose an image which will have all the three options upload, choose from media library or add from a link and will keep the placeholder abstract.

## Testing Instructions
1. Add a columns block or grid block. Choose 6+ columns or grid in a row and add an image block.
2. Now the choose an image option will appear in BlockControls Toolbar.  

## Screenshots or screencast

For Large Size
<img width="676" alt="image" src="https://github.com/user-attachments/assets/2161cd0c-5a1e-4f2d-8511-a8e928825661">


For Medium Size
<img width="534" alt="image" src="https://github.com/user-attachments/assets/a339a420-8e20-4a15-9335-4844df6beb04">

For small Size
<img width="483" alt="image" src="https://github.com/user-attachments/assets/1b88635f-df3b-452f-9fcf-72d6b26f45ec">
